bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/config/config2.xml

```diff
@@ -8,7 +8,6 @@
 		<parameter name="deviceIp" type="text" required="true">
 			<label>@text/thing-type.config.shelly.deviceIp.label</label>
 			<description>@text/thing-type.config.shelly.deviceIp.description</description>
-			<context>network-address</context>
```

Comment:
This is somewhat unexpected. Why remove this?

Reply:
The network-address context doesn't allow the format ip-address:port, which is now required to address secondary Shellys linked to a range extender device. Therefore I need a regular text field